packages/contentstack-branches/README.md

```diff
@@ -37,7 +37,7 @@ $ npm install -g @contentstack/cli-cm-branches
 $ csdx COMMAND
 running command...
 $ csdx (--version)
-@contentstack/cli-cm-branches/1.4.3 darwin-x64 node-v22.17.0
+@contentstack/cli-cm-branches/1.4.3 darwin-arm64 node-v22.14.0
```

Comment:
If there is no change in the plugin, you don't need to push the README files too.